### ISSUE TITLE
The screenshots are not generated

### ISSUE THREAD

**Issue by shaky** [sitespeedio/sitespeed.io#691]
Hi guys,
First of all I want to thank you for all the hard work. The tool is amazing!
Looks like the screenshots are not generated or my configuration is wrong.

Here's the command I'm running:
sitespeed.io -u URL -b chrome --profile mobile --screenshot --connection mobile3g -d 2

This is the info line:
info: OS: 'darwin 13.4.0', Node.js: 'v0.10.29', sitespeed.io: '3.6.3', PhantomJS: '1.9.8', java: '1.7.0_65', browsertime: '0.9.8'

Thanks for help!

**Comment by soulgalore** [comment]
Hi @shaky ,
ok, can you send me the logs from the run (in your result folder there's a sitespeed.io.log, browsertime.log and browsermobproxy.log) then I can see what's happening. 

My email: peter [AT] soulgalore [dot] com

Thanks for reporting! 
Peter

**Comment by shaky** [comment]
Hi @soulgalore 
Thanks for reply.
I have just emailed you the logs.
Cheers!
Raz

**Comment by jzoldak** [comment]
Hi @soulgalore - first echo'ing @shaky's thanks for your awesome project.
FWIW I'm also am unable to capture screenshots. 
Here's the relevant section from my sitespeed.io.log.
If I get some time tomorrow I'll try to dig into it a bit.
```
2015-08-11T19:11:13.395Z - info: [sitespeed] OS: 'linux 3.2.0-65-virtual', Node.js: 'v0.10.37', sitespeed.io: '3.8.0', PhantomJS: '1.9.8', java: '1.7.0_79', browsertime: '0.10.2'
[...]

2015-08-11T19:12:51.378Z - info: [sitespeed] Taking screenshot for <URL for my page under test>
2015-08-11T19:13:03.453Z - error: [sitespeed] Phantomjs output for '<URL for my page under test>' (stdout): SyntaxError: Unable to parse JSON string

  /usr/lib/node_modules/sitespeed.io/lib/headless/scripts/screenshot.js:32

2015-08-11T19:13:03.456Z - error: [sitespeed] Error taking screenshots for url '<URL for my page under test>': { [Error: Command failed: ] killed: true, code: null, signal: 'SIGTERM' }
```

**Comment by soulgalore** [comment]
Hi @jzoldak ,
thanks again :)

ah ok, good this seems like there's a bug, can you share me the URL so I can reproduce peter [at] soulgalore [dot] com. hmm tested now and if there's javascript error on the page it fails (tested with <code>-u http://www.aftonbladet.se -d 0 --screenshot</code>).

Let me fix that tomorrow. If your error seems to be something else, please drop me the URL :)

Best
Peter

**Comment by jzoldak** [comment]
Aha! Cool @soulgalore that's awesome you got that bug.

My setup is pretty convoluted as it requires both basic auth to get to the server and a logged-in session to get to the page. I think my problem it's something different, but I will make you a sandbox tomorrow if I can't figure it out. I believe I have a suspect for the root cause tho. :) I'll submit a PR and link it to this issue.

**Comment by soulgalore** [comment]
Hey @shaky ,
best case scenario this was fixed in https://github.com/sitespeedio/sitespeed.io/issues/725 
 Would you be able to test from the master branch?

Best
Peter

**Comment by soulgalore** [comment]
This is now released in 3.8.1. @shaky please re-open if this is not working for you.
